phys/module_surface_driver.F

```diff
@@ -3895,6 +3895,20 @@ SUBROUTINE surface_driver(                                         &
 
 
 !------------------------------------------------------------------
+        !For WRF-CTSM simulations, we would like the land model (CTSM)
+        ! to handle inland lake points.
+        ! Here, we are changing xland to include lake points, so that
+        ! CTSM can handle it. 
+         DO j=j_start(ij),j_end(ij)
+            DO i=i_start(ij),i_end(ij)
+                IF (lakemask(i,j).EQ.1. .AND. xland(i,j).GT.1.5 .AND. ht(i,j).GT.lake_min_elev) THEN
+                    WRITE( message,* ) 'WRF-CTSM: for great lakes  xland ==1'
+                    CALL wrf_message ( message )
+                    xland (i,j)  = 1  
```

Comment:
Is lakemask=1 for the Great Lakes? This only works if it is, but I thought it would be 0 since these are too large to qualify for the lake mask.
Also does CTSM really need xland=1 to handle the Great Lakes? Does it have its own internal lake model? Normally xland=1 is only for the land and lakemask is used for lakes.

Reply:
@negin513  still needs to answer my question here. Not sure if this is correct yet.

Reply:
I think our suggestion was to do this by not modifying xland and instead passing a temporary local parameter into the CTSM driver which is 1 for the lake points.

Reply:
It seems like lakemask=1 for the Great Lakes. I am attaching the plots from wrfout that I showed you in the previous meeting which shows Great Lakes in the lakemask:
![image](https://user-images.githubusercontent.com/17344536/110026610-13bac500-7cee-11eb-959a-4ee3136da619.png)

This shows the lakemask for wrfinput:
![image](https://user-images.githubusercontent.com/17344536/110026949-9774b180-7cee-11eb-93a5-e23f5208e62d.png)


I am putting these here only for the sake of future reference.

Reply:
> I think our suggestion was to do this by not modifying xland and instead passing a temporary local parameter into the CTSM driver which is 1 for the lake points.

@dudhia Correct! The suggestion by the WRF committee was that instead of directly changing `xland` variable, we should make a local variable called `ctsm_xland` which has 1 for the lake points. Then we pass `ctsm_xland` instead of `xland` to CTSM.